Xamarin.Essentials/MediaPicker/MediaPicker.uwp.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+
+namespace Xamarin.Essentials
+{
+    public static partial class MediaPicker
+    {
+        static readonly string[] defaultVideoFileTypes = new[] { ".mp4", ".wmv", ".avi" };
+        static readonly string[] defaultImageFileTypes = new[] { ".jpeg", ".jpg", ".png", ".gif", ".bmp" };
```

Comment:
We may need to evaluate this and see if we want more or makle it customizable. TIFF anyone?